lib/models/device.coffee

```diff
@@ -1345,6 +1345,45 @@ getDeviceModel = (deps, opts) ->
 						force: Boolean(options.force)
 		.asCallback(callback)
 
+	###*
+	# @summary Get the supervisor state on a device
+	# @name getSupervisorState
+	# @public
+	# @function
+	# @memberof balena.models.device
+	#
+	# @param {String|Number} uuidOrId - device uuid (string) or id (number)
+	# @returns {Promise}
+	#
+	# @example
+	# balena.models.device.getSupervisorState('7cf02a6');
+	#
+	# @example
+	# balena.models.device.getSupervisorState(123);
+	#
+	# @example
+	# balena.models.device.getSupervisorState('7cf02a6',
+	# function(error) {
+	# 	if (error) throw error;
+	# });
+	###
+	exports.getSupervisorState = (uuidOrId, callback) ->
+		callback = findCallback(arguments)
```

Comment:
We don't really need `findCallback`, since if the callback is provided, it will be the second argument.
```suggestion
```
The purpose of `findCallback`, is for the case that the method accepts optional arguments, which can make callback arg position vary, like in `device.getAll()`.

Reply:
Addressed in latest push